canopeum_frontend/src/utils/dateFormatter.ts

```diff
@@ -0,0 +1,11 @@
+export const formatDate = (dateIso: string): string => {
```

Comment:
Let's use the configurable, locale-aware date formatting that comes with javascript !
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString